src/Options.ts

```diff
@@ -70,12 +47,47 @@ export default class Options {
     readonly dry: boolean
   ) {}
 
+  private pluginLoader = new PluginLoader([
+    new FileSystemResolver(),
+    new PackageResolver()
+  ]);
+
+  private remotePluginLoader = new PluginLoader([
+    new AstExplorerResolver(),
+    new NetworkResolver()
```

Comment:
Eventually these lists might be configurable, but for now local plugins are loaded from either a direct file system reference or by package name and remote plugins are loaded either from astexplorer.net editor URLs or just a raw URL.